package.json

```diff
@@ -14,6 +14,7 @@
     "@jest/globals": "workspace:*",
     "@jest/test-utils": "workspace:*",
     "@tsconfig/node10": "^1.0.8",
+    "@tsd/typescript": "~4.1.5",
```

Comment:
The actual installed `typescript` version is `4.1.6`. Unfortunately `tsd` did not publish its own patch for `4.1.6`. The closes available is `4.1.5`. See: https://www.npmjs.com/package/@tsd/typescript?activeTab=versions